api/soap/mc_issue_api.php

```diff
@@ -1659,6 +1670,12 @@ function mci_issue_data_as_header_array( BugData $p_issue_data ) {
 		$t_issue['attachments_count'] = count( mci_issue_get_attachments( $p_issue_data->id ) );
 		$t_issue['notes_count'] = count( mci_issue_get_notes( $p_issue_data->id ) );
 
+		if( ( ON === config_get_global( 'public_urls' ) ) && isset( $p_issue_data->token ) ) {
```

Comment:
Is this attribute important enough to be in the header information for the list of issues?